app/models/listing_shape.rb

```diff
@@ -14,10 +14,12 @@
 #  sort_priority              :integer          default(0), not null
 #  created_at                 :datetime         not null
 #  updated_at                 :datetime         not null
+#  deleted                    :boolean          default(FALSE)
 #
 # Indexes
 #
 #  index_listing_shapes_on_community_id  (community_id)
+#  index_listing_shapes_on_deleted       (deleted)
```

Comment:
Needs validation if this index is actually used with some query since MySQL cannot combine indices.